source/jormungandr/jormungandr/realtime_schedule/timeo.py

```diff
@@ -132,9 +132,9 @@ def _get_next_passage_for_route_point(self, route_point, count=None, from_dt=Non
                                               .format(r.url))
             return None
 
-        return self._get_passages(r.json())
+        return self._get_passages(r.json(), route_point.fetch_line_uri())
 
-    def _get_passages(self, timeo_resp):
+    def _get_passages(self, timeo_resp, line_uri):
```

Comment:
minor: you may default line to None and avoid passing None in tests, or when not usefull